components/action-mgt/org.wso2.carbon.identity.action.management/src/main/java/org/wso2/carbon/identity/action/management/ActionManagementServiceImpl.java

```diff
@@ -140,6 +143,28 @@ public Action getActionByActionId(String actionId, String tenantDomain) throws A
         return CACHE_BACKED_DAO.getActionByActionId(actionId, IdentityTenantUtil.getTenantId(tenantDomain));
     }
 
+    @Override
+    public Action updateActionEndpointAuthentication(String actionType, String actionId, AuthType authentication,
+                                                     String tenantDomain) throws ActionMgtException {
+
+        if (LOG.isDebugEnabled()) {
+            LOG.debug("Updating Action endpoint authentication for Auth Type: " + authentication.getType().name() +
+                    " and Action Type: " + actionType + " and Action Id: " + actionId);
```

Comment:
This debug log doesn't convey the message well.
Better say,
Updating endpoint authentication of Action Type: {} Action ID: {}  to AuthType: {}

Reply:
FIxed with https://github.com/wso2/carbon-identity-framework/pull/5795/commits/4c40292ff2e65531731f8e68897c74215dae5279